content/project-organization/people-and-roles.md

```diff
@@ -29,11 +28,10 @@ The product council is in charge of product vision for the project, ensuring tha
 
 Current members:
 
-- Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon)) - **Council Lead**
+- Amine Ouanes ([@amine-O](https://github.com/amine-O)) - **Council Lead**
 - Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox))
 - Jean-François Viguier ([@jf-viguier](https://github.com/jf-viguier))
```

Comment:
```suggestion
- Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox)) - external contributor not affiliated with PrestaShop SA 
- Jean-François Viguier ([@jf-viguier](https://github.com/jf-viguier)) - external contributor not affiliated with PrestaShop SA 
```

@matks wdyt?

Reply:
No opinion on this topic, so okay for me